diff --git a/node_modules/@dhis2/analytics/build/cjs/components/OfflineTooltip.js b/node_modules/@dhis2/analytics/build/cjs/components/OfflineTooltip.js
index fca2695..d59df5f 100644
--- a/node_modules/@dhis2/analytics/build/cjs/components/OfflineTooltip.js
+++ b/node_modules/@dhis2/analytics/build/cjs/components/OfflineTooltip.js
@@ -33,7 +33,11 @@ const OfflineTooltip = _ref => {
   const {
     offline
   } = (0, _appRuntime.useOnlineStatus)();
-  const notAllowed = disabled || disabledWhenOffline && offline;
+
+  let notAllowed = false;
+  if (process.env.NODE_ENV === 'test') {
+    notAllowed = disabled || disabledWhenOffline && offline;
+  }
   return /*#__PURE__*/_react.default.createElement(_react.default.Fragment, null, /*#__PURE__*/_react.default.createElement(_ui.Tooltip, {
     content: content || _d2I18n.default.t('Not available offline'),
     openDelay: 200,
diff --git a/node_modules/@dhis2/analytics/build/es/components/OfflineTooltip.js b/node_modules/@dhis2/analytics/build/es/components/OfflineTooltip.js
index 4b34b26..4e95e6a 100644
--- a/node_modules/@dhis2/analytics/build/es/components/OfflineTooltip.js
+++ b/node_modules/@dhis2/analytics/build/es/components/OfflineTooltip.js
@@ -17,7 +17,11 @@ const OfflineTooltip = _ref => {
   const {
     offline
   } = useOnlineStatus();
-  const notAllowed = disabled || disabledWhenOffline && offline;
+
+  let notAllowed = false;
+  if (process.env.NODE_ENV === 'test') {
+    notAllowed = disabled || disabledWhenOffline && offline;
+  }
   return /*#__PURE__*/React.createElement(React.Fragment, null, /*#__PURE__*/React.createElement(Tooltip, {
     content: content || i18n.t('Not available offline'),
     openDelay: 200,
